thefuck/rules/git_remote_delete.py

```diff
@@ -0,0 +1,15 @@
+from thefuck.utils import replace_argument
+from thefuck.specific.git import git_support
+
+
+@git_support
+def match(command):
+    return "git remote delete" in command.script
+
+
+@git_support
+def get_new_command(command):
+    return replace_argument(command.script, "delete", "remove")
```

Comment:
It doesn't work for `git remote delete delete`. Better use `re.sub` directly instead of `replace_argument`.